docker-compose.yml

```diff
@@ -17,7 +18,7 @@ services:
       - assets_volume:/app/experimenter/static/assets/
       - static_volume:/app/experimenter/served/
       - /app/.cache
-    command: bash -c "/app/bin/wait-for-it.sh db:5432 -- python /app/manage.py collectstatic --noinput;python /app/manage.py runserver 0:7001"
+    command: bash -c "/app/bin/wait-for-it.sh kinto:8888 -- python bin/setup_kinto.py;/app/bin/wait-for-it.sh db:5432 -- python /app/manage.py collectstatic --noinput;python /app/manage.py runserver 0:7001"
```

Comment:
This line gets quite dense now, shall we switch to the exec form? Or perhaps put them into a startup script?